SA/design_concept/content/results/results_poc.typ

```diff
@@ -0,0 +1,21 @@
+= Proof of Concept
+The project produced a proof of concept application, that implements the design concept described in @design_proposal_iteration_2 and @design_specification.
+While the application doesn't implement all concepts required to learn functional programming, like the construction of custom data types, the application shows that the concept for function composition works and is easy to use.
+This means that the main goal of the project was achieved.
+
+To make the application ready to be used in a classroom, the use cases that weren't achieved during this project, as can be seen in @requirement_validation, need to be implemented.
+
+In addition, there is some potential for improvement in the current implementation of the PoC.
+The choice for Threepenny as the UI framework was made primarily due to the given time constraints and the expectation that Threepenny would fast progress while implementing the PoC (which proved to be true).
+A, propably GHCJS based, UI technology that doesn't require a local webserver would be better suited to implement a full version of VisualFP.
+A different UI framework may also provide better support for functional reactive programming, which is expected to make the UI implementation more concise.
+
+== macOS Electron App
+For the project submission, the PoC Electron application was packaged for Windows, Linux & macOS.
+The macOS app brought up an unforseen issue:
```

Comment:
```suggestion
The macOS app brought up an unforeseen issue:
```

Reply:
This issue isn't solved yet

Reply:
😮 I'm petty sure that is corrected that, but it seems like it got lost somehow.